tools/disass/ida/disass.py

```diff
@@ -377,11 +464,36 @@ def get_called_subroutines(sub):
       called_sub_eas.add(get_direct_branch_target(term_inst.ea))
   return called_sub_eas
 
+# Python 2.7 doesn't range on longs
+def lrange(start, end):
+  num = long(start)
+  while num < end:
+    yield num
+    num += 1L
+
+# source: http://reverseengineering.stackexchange.com/a/13472/4197
```

Comment:
Can you add a comment describing what this function does, and why. The purpose of the code on the RE stack exchange is not the same as what this code is doing.